src/page-merger-spec.js

```diff
@@ -0,0 +1,62 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+import PageMerger from 'src/page-merger';
+import PageNumberPaginator from 'src/paginators/page-number-paginator';
+import * as request from 'src/helpers/request-spy';
+
+
+const {expect} = chai;
+
+chai.use(chaiAsPromised);
+
+describe('drf-paginator', function() {
+  afterEach(function() {
+    request.spy.reset();
+  });
+
+  describe('PageMerger', function() {
+    describe('merge', function() {
+      const pageCount = 10;
+      let pageMerger;
+
+      beforeEach(function() {
+        const paginator = new PageNumberPaginator(request.spy);
+        pageMerger = new PageMerger(paginator);
+      });
+
+      it('returns a promise that successfully resolves', function() {
+        const promsie = pageMerger.merge(1, pageCount);
```

Comment:
Promsie? I'm imagining Jar Jar Binks now asking Padma to the prom now. "Yousa go to the promsie withsa me?"